app/src/main/java/org/p2p/wallet/datastore/UserDataStoreEditorImpl.kt

```diff
@@ -0,0 +1,38 @@
+package org.p2p.wallet.datastore
+
+import androidx.datastore.preferences.core.MutablePreferences
+import androidx.datastore.preferences.core.booleanPreferencesKey
+import androidx.datastore.preferences.core.doublePreferencesKey
+import androidx.datastore.preferences.core.intPreferencesKey
+import androidx.datastore.preferences.core.stringPreferencesKey
+
+class UserDataStoreEditorImpl(private val preferences: MutablePreferences) : UserDataStore.Editor {
```

Comment:
is it just facade for DataStore, or is it datastore specially for prefs.xml or account_prefs.xml?

Reply:
yeah it's just library behind interface facade ))

Reply:
I suggest naming it as Facade then.
and why User and not App for example? AppDataStore

Reply:
Let's discuss the naming in android sync. 
Not sure if Facade is a good name